### DISCUSSION TITLE
README.md not showing any text in NVIDIA/open-gpu-kernel-modules

### DISCUSSION THREAD

**Discussion by TheRealOne78** [community/community#19967] [Mobile]
As the title says, for some reason github mobile for android is not showing any type of text from README.md in NVIDIA/open-gpu-kernel-modules, unlike the browser which shows everything needed to be shown.

I noticed this issue a few beta releases ago and it’s still not showing any text.

This is how NVIDIA/open-gpu-kernel-modules’s README.md is showing in a normal browser:
<img src="https://aws1.discourse-cdn.com/github/original/3X/b/2/b2950005e87f2a361379a0195fcb116b69fd10b8.png">

and this is how Github for android is rendering README.md:
<img src="https://aws1.discourse-cdn.com/github/original/3X/2/5/25276d06f16547e8c00ca192ace2d9dfbf4fa22c.png">

However other repos are rendering README.md just fine just like the photo below, the only repo I found with this issue would be NVIDIA/open-gpu-kernel-modules.
<img src="https://aws1.discourse-cdn.com/github/original/3X/d/7/d7f52d701230d6929230cdf33ceec8076e39991f.png">

_*This issue is ported from https://github.community/t/readme-md-not-showing-any-text-in-nvidia-open-gpu-kernel-modules/257980_

**Comment by TheRealOne78** [comment]
Originally, this issue was made a while ago, by some user's recommandation I ported it here without noticing now README.md renders.

![Screenshot_20220704-131033_GitHub.png](https://user-images.githubusercontent.com/53640879/177133728-fd3d308a-355b-4323-b712-c3a43e5017bb.png)

**Comment by diegoleondev** [comment]
You can activate the "'desktop mode' in the smartphone's browser, in order to access the content.

**Reply by TheRealOne78** [reply]
Yes but the bug would still be existent in the android app.
Anyway, the bug seems to be fixed so it's okay now.

**Reply by Rootie2shoes** [reply]
# https://www.google.com/search?q=asp+net+example&tbm=isch&bih=779&biw=412&client=ms-android-samsung&prmd=bnvi&hl=en&sa=X&ved=2ahUKEwiIrcuL59z4AhWDnVMKHRmfD54QrNwCKAB6BQgBEIYC

**Reply by TheRealOne78** [reply]
I am not exactly sure what that URL is suposed to mean, but I have a slight suspicion you don't get what I mean.
This discussion is about the android application of github, not the website in a mobile browser: https://play.google.com/store/apps/details?id=com.github.android
This issue was originally made to report to github about this bug existent in the app, not the mobile web version of github.

**Comment by elisealix22** [comment]
@TheRealOne78 thank you for reporting this! I believe this is related to an open issue we are tracking with large READMEs freezing up. I added a comment on our tracking issue with the repository you reported (thank you for providing an example!) to help debug. Let us know if you run into any other issues!
